video_prediction/lstm_ops.py

```diff
@@ -99,7 +97,9 @@ def basic_conv_lstm_cell(inputs,
                             scope='Gates')
 
     # i = input_gate, j = new_input, f = forget_gate, o = output_gate
-    i, j, f, o = tf.split(axis=3, num_or_size_splits=4, value=i_j_f_o)
+
+    i, j, f, o = tf.split(i_j_f_o, 4,3 )
```

Comment:
Seems necessary to me, according to the api https://www.tensorflow.org/api_docs/python/tf/split, and if I use the old style it raise IndexError:list index out of range. The tf.__version__ in my conputer is  1.0.0, is it required the mater version from github? Or maybe the api document haven't update yet.